cosmos/hub/dym_hub/dym_hub.go

```diff
@@ -92,3 +92,15 @@ func (c *DymHub) SetRollApp(rollApp ibc.RollApp) {
 func (c *DymHub) GetRollApp() ibc.RollApp {
 	return c.rollApp
 }
+
+func (c *DymHub) FullfillDemandOrder(ctx context.Context,
+	node *cosmos.Node,
```

Comment:
Does the DymHub using it own node or using other chain node? If using the own node. we could use some thing like `c.GetNode().ExecTx`